app.rb

```diff
@@ -4,17 +4,57 @@
 
 # First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
   # Explain how the data is structured
+  It's categorized by apartments and tenants. It has string value sand numerical values.
```

Comment:
That's correct, but I'd flesh this out a bit by saying that it's an array of hashes, where each hash represents either e.g. an apartment.